addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -18,6 +18,32 @@
         </xpath>
     </template>
 
+    <template id="product_cart_lines" inherit_id="website_sale.cart_lines" active="True">
+        <div name="o_wsale_cart_line_button_container" position="inside">
+            <span class="d-none d-md-inline-block ms-1">
+                <a
+                    href="#"
+                    class="small o_add_wishlist js_delete_product px-2"
+                    style="border-left: 1px solid"
```

Comment:
I'm not a designer but I think we should make this border the same grey as the one separating two product lines. The current one is the same purple as the buttons.

Reply:
Done :heavy_check_mark: